rust/Cargo.lock

```diff
@@ -0,0 +1,21 @@
+[root]
+name = "suricata"
+version = "4.0.0-dev"
```

Comment:
Would there be a way to make this dynamic? E.g. be set based on the configure version? Perhaps it would mean this file would be checked in as a .in file?